Sentry.sln

```diff
@@ -276,6 +275,7 @@ Global
 		{4E0DC405-C372-4396-A5DF-F6AA108DA01C}.Release|Any CPU.Build.0 = Release|Any CPU
 		{9B175EC8-6B64-4345-A158-091CB8876077}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{9B175EC8-6B64-4345-A158-091CB8876077}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{9B175EC8-6B64-4345-A158-091CB8876077}.Debug|Any CPU.Deploy.0 = Debug|Any CPU
```

Comment:
This looks like the only relevant change... all the rest of it is just messing with GUIDs. Could we possibly make this change manually rather than using Visual Studio, to avoid all the unnecessary messing about that it's done with the solution file?

Eventually we should probably move to slnx files (once the [tooling supports these](https://github.com/dotnet/sdk/issues/40913))

Reply:
it's a generated file so not sure it's worth doing any manual changes. Whatever VS does, we just push and merge